src/components/cc-zone-picker/cc-zone-picker.js

```diff
@@ -35,13 +38,31 @@ export class CcZonePicker extends CcFormControlElement {
   constructor() {
     super();
 
-    /** @type {ZonesSections} array of zones sections */
-    this.zonesSections = [];
+    /** @type {boolean} whether all the form controls should be disabled */
+    this.disabled = false;
+
+    /** @type {boolean} whether all the form controls should be readonly */
+    this.readonly = false;
 
     /** @type {string} current selected zone code */
     this.value = null;
+
+    /** @type {ZonesSections} array of zones sections */
+    this.zonesSections = [];
   }
 
+  /**
+   * @param {CcZonePickerPropertyValues} changedProperties
+   */
+  willUpdate(changedProperties) {
+    if (changedProperties.has('value') && this.value == null) {
+      this.value = this.zonesSections?.[0].zones[0].code;
```

Comment:
fix: if user doesn't set any `zonesSections`, this will break because `this.zonesSections` will have a length of zero. We should check if `this.zonesSections` has got at least one item. (and this is the same for the `zones` array inside the first zone section)